infra/containers.yaml

```diff
@@ -6,6 +6,11 @@ Mappings:
       image: public.ecr.aws/t1q6b4h2/universal-application-tool:prod
     staging:
       image: public.ecr.aws/t1q6b4h2/universal-application-tool:latest
+  S3TaskRoles:
```

Comment:
This is a heck of a nitpick - but would you mind putting this at the root level and passing it in as an argument to each of these substacks, so that we can have it in only one place?

Reply:
Yeah, agreed that's better. Done.